js/views/file_input_view.js

```diff
@@ -202,13 +204,15 @@
     },
 
     clearAttachments() {
-      this.attachments.forEach(attachment => {
+      this.attachments.forEach(async attachment => {
```

Comment:
So, because we're not using `Promise.all()` here, any rejections here will be top-level exceptions. I think it would be better to be explicit about what should happen if an error happens in any of these.